.github/drafts/16298/(Rehaul)-Docker-Outline.md

```diff
@@ -0,0 +1,65 @@
+
+## Introduction
+ * Docker Intro
+ * Why Use Docker 
+
+## Prerequisites 
+The versions and files required to install Docker will be covered in this section.
+//cross-link to docker docs
+* Docker
+* Docker-Compose 
+
+### Steps to check the installed versions
+
+## Get started with Docker compose (H1)
+In this section, we will discuss how to configure docker-compose with a .yml file and with the curl command. 
+### Introduction 
+* Understanding docker-compose files
+
+### Docker-compose configuration (H2)
+* .yml file setup
+
+### Start your containers (H2)
+We will run the containers in this section and split it into two sections (CE and BE)
+* Community Edition (H3)
+  * Configuration
+
+* Business Edition (H3)
+  * Configuration
+
+### Setting up the app/Getting Started  (H2)
+What users can do after container installation will be covered in this section. (Example: explanations on how to set up the first app, details on the admin settings page, etc.)
+* Set up (with video example)
+* Admin setting page info
+* Deploy
+
+### Updating Appsmith (H2)
```

Comment:
Updates and Auto-Updates sections should be after the section on installing Appsmith using Docker run

Reply:
If we create a new section for updates and auto-updates, we will have to create 4 sub-sections: updates-with docker-compose(CE and BE), updates (without docker-compose) (CE and BE), and same with auto-updates.

or we can add updates and auto-updates inside with docker-compose and without docker-compose sections. @jnikhila @Pranay105 your thoughts?

Reply:
@sharat87 / @sum35h - Could you please guide us here?

Reply:
I think notes on updating can be togather next to installation instructions. People would want instructions for installing and updating with `docker`, or installing and updating with `docker-compose`. But, I'd rarely need instructions for installing with `docker` and `docker-compose` togather. So I think it makes sense to structure that way?